docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+volumes:
+  nextcloud:
+  db:
+
+services:
+
+  db:
+     image: mariadb:10.6
+     restart: always
+     command: --transaction-isolation=READ-COMMITTED --log-bin=binlog --binlog-format=ROW
+     volumes:
+       - db:/var/lib/mysql
+     environment:
+       - MYSQL_ROOT_PASSWORD=root
+       - MYSQL_PASSWORD=sqlusr
+       - MYSQL_DATABASE=nextcloud
+       - MYSQL_USER=nextcloud
+
+  app:
+    image: nextcloud
+    restart: always
+    ports:
+      - 8080:80
+    links:
+      - db
+    volumes:
+      - nextcloud:/var/www/html
+    environment:
+      - MYSQL_PASSWORD=sqlusr
+      - MYSQL_DATABASE=nextcloud
+      - MYSQL_USER=nextcloud
+      - MYSQL_HOST=db
+      - NEXTCLOUD_ADMIN_USER:testuser
+      - NEXTCLOUD_ADMIN_PASSWORD:1234
```

Comment:
Lets maybe move these things into a .env file and specify in the README.md what keys the .env should have

Reply:
My SQL host should still be hardcoded but the rest I agree, it's ppbl. cleaner to do that in an file.